src/content/app/genome-browser/services/genome-browser-service/genomeBrowserCommands.ts

```diff
@@ -53,8 +56,6 @@ export const setBrowserLocation = (payload: {
       focusType: focus.type
     });
   }
-
-  genomeBrowser.goto(start, end);
```

Comment:
Moved the goto instruction before the setting the focus, because otherwise the genome browser does this:

https://github.com/Ensembl/ensembl-client/assets/6834224/48b85e02-4f7b-4cd8-aa9d-53529d1becb5

I have no idea why this should have made a difference. Bloody genome browser.